invisible_cities/io/dst_io.py

```diff
@@ -1,13 +1,29 @@
 import tables as tb
 import pandas as pd
-
+from .. core.exceptions import UnknownDST
 
 def load_dst(filename, group, node):
     with tb.open_file(filename) as h5in:
-        table = getattr(getattr(h5in.root, group), node).read()
+        try:
+            table = getattr(getattr(h5in.root, group), node).read()
+        except:
```

Comment:
`except` without an exception specification is rarely the correct thing to do. You are saying, "I don't care what message is being sent here, whatever it is I know that the way to deal with it is to print a message to standard out and to translate the exception to `UnknownDST`.

What if the message is "Incoming nuclear missile, run to the nuclear shelters right now". Who cares, let's print a message to standard out and translate it to `UnknownDST`.

Please handle the exceptions that you are expecting, and let all others propagate down the stack to whoever *really* knows how to deal with them.